### PR TITLE
feat(ITKHelper): Add conversion functions for PolyData

### PR DESCRIPTION
**Pull request by thewtex**
Also add example.

### CODE REVIEW

File: Sources/Common/DataModel/ITKHelper/index.js [hunk 1]
```diff
@@ -238,7 +250,320 @@ function convertVtkToItkImage(vtkImage, copyData = false) {
   return itkImage;
 }
 
+/**
+ * Converts an itk-wasm Image to a vtk.js vtkImageData.
```
Comment (finetjul) [inline Sources/Common/DataModel/ITKHelper/index.js:254]: itk-wasm Image -> itk-wasm Mesh ?

File: Sources/Common/DataModel/ITKHelper/index.js [hunk 2]
```diff
@@ -238,7 +250,320 @@ function convertVtkToItkImage(vtkImage, copyData = false) {
...
+        return null;
+    }
+  }
+  return polyData;
+}
+
+/**
+ * Converts a vtk.js vtkPolyData to an itk-wasm PolyData.
+ *
+ * Requires a vtk.js vtkImageData as input.
```
Comment (finetjul) [inline Sources/Common/DataModel/ITKHelper/index.js:485]: vtkImageData -> vtkPolyData

File: Sources/Common/DataModel/ITKHelper/index.js (hunk 1 above)
Comment (thewtex) [reply Sources/Common/DataModel/ITKHelper/index.js:254]: Yes, it should be PolyData.

File: Sources/Common/DataModel/ITKHelper/index.js (hunk 2 above)
Comment (thewtex) [reply Sources/Common/DataModel/ITKHelper/index.js:485]: :+1:

File: Sources/Common/DataModel/ITKHelper/index.js [hunk 3]
```diff
@@ -238,7 +250,320 @@ function convertVtkToItkImage(vtkImage, copyData = false) {
...
+      name: '_lines',
+      numberOfComponents: 1,
+      size: itkPolyData.linesBufferSize,
+      dataType: 'Uint32Array',
+      buffer: itkPolyData.lines.buffer,
+      values: itkPolyData.lines,
+    },
+    polys: {
+      vtkClass: 'vtkCellArray',
+      name: '_lines',
```
Comment (floryst) [inline Sources/Common/DataModel/ITKHelper/index.js:321]: Name to _polys

File: Sources/Common/DataModel/ITKHelper/index.js [hunk 4]
```diff
@@ -238,7 +250,320 @@ function convertVtkToItkImage(vtkImage, copyData = false) {
...
+      numberOfComponents: 1,
+      size: itkPolyData.polygonsBufferSize,
+      dataType: 'Uint32Array',
+      buffer: itkPolyData.polygons.buffer,
+      values: itkPolyData.polygons,
+    },
+    strips: {
+      vtkClass: 'vtkCellArray',
+      name: '_lines',
```
Comment (floryst) [inline Sources/Common/DataModel/ITKHelper/index.js:330]: Name to _strips

File: Sources/Common/DataModel/ITKHelper/index.js (hunk 3 above)
Comment (thewtex) [reply Sources/Common/DataModel/ITKHelper/index.js:321]: :+1:

File: Sources/Common/DataModel/ITKHelper/index.js (hunk 4 above)
Comment (thewtex) [reply Sources/Common/DataModel/ITKHelper/index.js:330]: :+1:

**Pull request merged**